packages/cardano-services/test/data-mocks/ledger.ts

```diff
@@ -2,351 +2,22 @@ export const parameters = {
-      'quotientInteger-cpu-arguments-constant': 196_500,
-      'quotientInteger-cpu-arguments-model-arguments-intercept': 453_240,
-      'quotientInteger-cpu-arguments-model-arguments-slope': 220,
-      'quotientInteger-memory-arguments-intercept': 0,
-      'quotientInteger-memory-arguments-minimum': 1,
-      'quotientInteger-memory-arguments-slope': 1,
-      'remainderInteger-cpu-arguments-constant': 196_500,
-      'remainderInteger-cpu-arguments-model-arguments-intercept': 453_240,
-      'remainderInteger-cpu-arguments-model-arguments-slope': 220,
-      'remainderInteger-memory-arguments-intercept': 0,
-      'remainderInteger-memory-arguments-minimum': 1,
-      'remainderInteger-memory-arguments-slope': 1,
-      'serialiseData-cpu-arguments-intercept': 1_159_724,
-      'serialiseData-cpu-arguments-slope': 392_670,
-      'serialiseData-memory-arguments-intercept': 0,
-      'serialiseData-memory-arguments-slope': 2,
-      'sha2_256-cpu-arguments-intercept': 806_990,
-      'sha2_256-cpu-arguments-slope': 30_482,
-      'sha2_256-memory-arguments': 4,
-      'sha3_256-cpu-arguments-intercept': 1_927_926,
-      'sha3_256-cpu-arguments-slope': 82_523,
-      'sha3_256-memory-arguments': 4,
-      'sliceByteString-cpu-arguments-intercept': 265_318,
-      'sliceByteString-cpu-arguments-slope': 0,
-      'sliceByteString-memory-arguments-intercept': 4,
-      'sliceByteString-memory-arguments-slope': 0,
-      'sndPair-cpu-arguments': 85_931,
-      'sndPair-memory-arguments': 32,
-      'subtractInteger-cpu-arguments-intercept': 205_665,
-      'subtractInteger-cpu-arguments-slope': 812,
-      'subtractInteger-memory-arguments-intercept': 1,
-      'subtractInteger-memory-arguments-slope': 1,
-      'tailList-cpu-arguments': 41_182,
-      'tailList-memory-arguments': 32,
-      'trace-cpu-arguments': 212_342,
-      'trace-memory-arguments': 32,
-      'unBData-cpu-arguments': 31_220,
-      'unBData-memory-arguments': 32,
-      'unConstrData-cpu-arguments': 32_696,
-      'unConstrData-memory-arguments': 32,
-      'unIData-cpu-arguments': 43_357,
-      'unIData-memory-arguments': 32,
-      'unListData-cpu-arguments': 32_247,
-      'unListData-memory-arguments': 32,
-      'unMapData-cpu-arguments': 38_314,
-      'unMapData-memory-arguments': 32,
-      'verifyEcdsaSecp256k1Signature-cpu-arguments': 35_892_428,
-      'verifyEcdsaSecp256k1Signature-memory-arguments': 10,
-      'verifyEd25519Signature-cpu-arguments-intercept': 9_462_713,
-      'verifyEd25519Signature-cpu-arguments-slope': 1021,
-      'verifyEd25519Signature-memory-arguments': 10,
-      'verifySchnorrSecp256k1Signature-cpu-arguments-intercept': 38_887_044,
-      'verifySchnorrSecp256k1Signature-cpu-arguments-slope': 32_947,
-      'verifySchnorrSecp256k1Signature-memory-arguments': 10
-    }
+    PlutusV1: [
+      197_209, 0, 1, 1, 396_231, 621, 0, 1, 150_000, 1000, 0, 1, 150_000, 32, 2_477_736, 29_175, 4, 29_773, 100, 29_773,
```

Comment:
Doesn't this require a change in mapper? :thinking:

Reply:
No. The mapper takes whatever db-sync has in the costs json, but does not attempt to change the contents.